docs/reference/release-notes/highlights-7.0.0.asciidoc

```diff
@@ -12,6 +12,110 @@ See also <<breaking-changes-7.0,Breaking changes>> and
 //NOTE: The notable-highlights tagged regions are re-used in the
 //Installation and Upgrade Guide
 
-// tag::notable-highlights[]
+//tag::notable-highlights[]
+=== Adaptive replica selection enabled by default
 
-// end::notable-highlights[]
\ No newline at end of file
+Words.
+//end::notable-highlights[]
+
+//tag::notable-highlights[]
+=== Skip shard refreshes if a shard is "search idle"
+
+Words.
+//end::notable-highlights[]
+
+//tag::notable-highlights[]
+=== Default to one shard
+
+Words.
+//end::notable-highlights[]
+
+//tag::notable-highlights[]
+=== Lucene 8
+
+Words.
+//end::notable-highlights[]
+
+//tag::notable-highlights[]
+=== Introduce the ability to minimize round-trips in {ccs}
+
+Words.
+//end::notable-highlights[]
+
+//tag::notable-highlights[]
+=== New cluster coordination implementation
+
+Words.
+//end::notable-highlights[]
+
+//tag::notable-highlights[]
+=== Real memory circuit breaker
+
+Words.
+//end::notable-highlights[]
+
+//tag::notable-highlights[]
+=== {ccr} is production-ready
+
+Words.
+//end::notable-highlights[]
+
+//tag::notable-highlights[]
+=== {ilm} is production-ready
+
+Words.
+//end::notable-highlights[]
+
+//tag::notable-highlights[]
+=== SQL is production-ready
```

Comment:
SQL was made GA in 6.7.0, do we need to mention it for 7.0 too?

Reply:
This was answered in another comment here: https://github.com/elastic/elasticsearch/pull/40481#issuecomment-476916583